tiatoolbox/tools/registration/prealignment.py

```diff
@@ -0,0 +1,96 @@
+import cv2
+import numpy as np
+import scipy.ndimage as ndi
+from skimage import exposure
+from skimage.util import img_as_float
+
+from tiatoolbox.utils.metrics import dice
+
+
+def prealignment(fixed_img, moving_img, fixed_mask, moving_mask, rotation_steps=10):
+    """Coarse registration of an image pair.
+
+    This function performs rough alignment of a moving image with respect
+    to a fixed image.
+
+    Args:
+            fixed_img (:class:`numpy.ndarray`):
+                    A grayscale fixed image.
+            moving_img (:class:`numpy.ndarray`):
+                    A grayscale moving image.
+            fixed_mask (:class:`numpy.ndarray`):
+                    A binary tissue mask for the fixed image.
+            moving_mask (:class:`numpy.ndarray`):
+                    A binary tissue mask for the moving image.
+
+    Returns:
+            :class:`numpy.ndarray`:
+                    A transform matrix.
+
+    """
+    if len(fixed_mask.shape) != 2:
+        fixed_mask = fixed_mask[:, :, 0]
+    if len(moving_mask.shape) != 2:
+        moving_mask = moving_mask[:, :, 0]
+
+    fixed_img, moving_img = np.squeeze(fixed_img), np.squeeze(moving_img)
```

Comment:
```suggestion
    fixed_img = np.squeeze(fixed_img)
    moving_img = np.squeeze(moving_img)
```

For readability and clarity

Reply:
Fixed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/427/commits/e2823eec36a84dc1f4bb6ffded110bd78be0f687